### ISSUE TITLE
Misplaced comma error for miscellaneous.csv

### ISSUE THREAD

**Issue by SimpleSandman** [FabulousCupcake/umamusume-db-translate#77]
As GitHub shows, the comma in the middle of the statement, `"8 - Profile, Dormitory"`, breaks comma-delimited convention. I assume removing the comma is what's desired since this represents a header row for the data grouped below it.

Ref: https://github.com/FabulousCupcake/umamusume-db-translate/blob/master/src/data/miscellaneous.csv#L3

**Comment by FabulousCupcake** [comment]
> I assume removing the comma is what's desired since this represents a header row for the data grouped below it.

This is correct, csv doesn't allow comments and the [`csvtojson.jq`](https://github.com/FabulousCupcake/umamusume-db-translate/blob/master/src/csvtojson.jq) looks for lines with two items, so they are ignored and essentially is a hacky way to add comments.

Still would be nice if we could use github's nice csv view for them though, without polluting `data.json` (or maybe it's fine)
